xilem_core/src/message.rs

```diff
@@ -17,6 +17,8 @@ pub enum MessageResult<Action> {
     /// This allows for sub-sections of your app to use an elm-like architecture
     Action(Action),
     // TODO: What does this mean?
+    /// This message's handler needs a rebuild to happen.
+    /// The exact semantics of this method haven't been determined.
```

Comment:
Well, it has been, but that was when we had async in xilem, it was used when a future has completed, sent a message and in `View::message` a rebuild was requested (e.g. when a list view entry in the async list has finished loading)

Reply:
But my question is how is that different from the implied result of an action handler at the root?

The semantics here are clearly not very principled

Reply:
You mean via `MessageResult:Action`?

Reply:
Yeah, exactly.